pages/linux/dex.md

```diff
@@ -0,0 +1,32 @@
+# dex
+
+> DesktopEntry Execution is a program to generate and execute DesktopEntry files of the Application type.
+> More information: <https://github.com/jceb/dex>.
+
+- Perform an autostart/execute all programs in the autostart folders:
+
+`dex --autostart`
+
+- Perform an autostart/execute all programs in the specified folders:
+
+`dex --autostart --search-paths {{path/to/directory}}:{{path/to/directory}}`
+
+- Preview the programs would be executed in a regular autostart:
+
+`dex --autostart --dry-run`
+
+- Preview the programs would be executed in a GNOME specific autostart:
+
+`dex --autostart --dry-run --environment GNOME`
+
+- Preview the value of DesktopEntry property Name:
+
+`dex --property Name {{path/to/file}}`
```

Comment:
```suggestion
`dex --property {{Name}} {{path/to/file}}`
```

Reply:
No, `Name` is a property to filter on.

Reply:
@Waples we should still put it in `{{token}}` syntax.

Reply:
Then a change to `{{property_name}}` or `{{property}}` is more suitable.

Reply:
We do that in other pages too. Even when we specify an example, we use `{{ }}`, to indicate that it can be replaced with something else.

Reply:
Fair enough :+1:

Reply:
https://github.com/tldr-pages/tldr/pull/5752#discussion_r614311435